server/schedule/operator/step.go

```diff
@@ -220,12 +231,13 @@ func (pl PromoteLearner) Influence(opInfluence OpInfluence, region *core.RegionI
 
 // RemovePeer is an OpStep that removes a region peer.
 type RemovePeer struct {
-	FromStore uint64
+	FromStore, PeerID uint64
 }
 
-// ConfVerChanged returns true if the conf version has been changed by this step
-func (rp RemovePeer) ConfVerChanged(region *core.RegionInfo) bool {
-	return region.GetStorePeer(rp.FromStore) == nil
+// ConfVerChanged returns the delta value for version increased by this step.
+func (rp RemovePeer) ConfVerChanged(region *core.RegionInfo) uint64 {
+	id := region.GetStorePeer(rp.FromStore).GetId()
+	return typeutil.BoolToUint64(id == 0 || (rp.PeerID != 0 && id != rp.PeerID))
```

Comment:
Why?

Reply:
Add some description.

Reply:
When will `PeerID` be set and when not set?

Reply:
It will not be set now, and the `builder` code will be updated later to use this new logic.